assets/app/styles/_project-menu.less

```diff
@@ -104,6 +101,12 @@
   }
 }
 
+.navbar-project-menu .bootstrap-select.btn-group .btn .filter-option {
+  max-width: 200px; // Width at 320px
+  min-width: 100%; // Now make menu full width and truncate at mobile
```

Comment:
Curious about that as well.   Not sure if we need 5 selectors to hit the `.filter-option`?